### ISSUE TITLE
error in calculating the TI error?

### ISSUE THREAD

**Issue by shuail** [MobleyLab/alchemical-analysis#102]
Hi, 
For TI only simulation, the calculation exits with "WARNING: Found all 0 in input data", and in the alchemical_analysis.py there is one line checking of all zeros. So I wonder if this line below:
_all_zeros = not numpy.any(dhdlt) and not numpy.any(u_klt)_
should be changed to:
_all_zeros = not numpy.any(dhdlt) or not numpy.any(u_klt)_
so when only dhdlt exists, the code still calculate the variance of TI, make plots etc?

**Comment by davidlmobley** [comment]
That sounds right. Do a PR? Thanks for pointing this out.

**Comment by shuail** [comment]
Great, yes, I can make a pull request about this. Could you give me the permission to commit? Thx!

**Comment by davidlmobley** [comment]
Done.

**Comment by shuail** [comment]
Hi David, thanks for the permission, I made a pull request for this issue and tested on my dataset. I realized that there is a typo in the first comments in this string. 

To be clear, the original code is _all_zeros = not numpy.any(dhdlt) or not numpy.any(u_klt)_ and the update changes this line to _all_zeros = not numpy.any(dhdlt) and not numpy.any(u_klt)_. 

The logic in the previous comments stays. For the TI only dataset, in the updated code numpy.any(dhdlt) = True, and numpy.any(u_klt) = False, and so all_zeros = not numpy.any(dhdlt) **and** not numpy.any(u_klt) will be False which is what we want -- we need the all_zeros to be False to avoid the exits. Sorry for the confusing in the previous comments...
